langchain4j-opensearch/src/main/java/dev/langchain4j/store/embedding/opensearch/OpenSearchEmbeddingStore.java

```diff
@@ -138,6 +138,20 @@ public OpenSearchEmbeddingStore(String serverUrl,
         this.indexName = ensureNotNull(indexName, "indexName");
     }
 
+    /**
+     * Creates an instance of OpenSearchEmbeddingStore to connect with
+     * OpenSearch clusters running as a fully managed service at AWS.
+     *
+     * @param openSearchClient OpenSearch client provided
+     * @param indexName   OpenSearch index name (optional). Default value: "default"
```

Comment:
the comment is wrong (also in other ctors), `indexName` is mandatory (unless  created via builder), would be awesome if you could fix it

Reply:
changed as well. no need to argue about indexName being set in builder 😄